packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -461,14 +470,16 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
         //
         // Explicit checks:
         // - The game must not have already been initialized.
-        // - An output root cannot be proposed at or before the genesis block.
+        // - An output root that is more than 1 day old cannot be proposed.
 
         // INVARIANT: The game must not have already been initialized.
         if (initialized) revert AlreadyInitialized();
 
-        // Do not allow the game to be initialized if the root claim corresponds to a block at or before the
-        // configured genesis block number.
-        if (l2BlockNumber() <= GENESIS_BLOCK_NUMBER) revert UnexpectedRootClaim(rootClaim());
+        // Do not allow the game to be initialized if the root claim corresponds to an L2 block that is more than
+        // 1 day old. Note that this check prevents output roots from being disputed until a chain has been live for
+        // at least 1 day.
+        uint256 expectedL2HeadNumber = (block.timestamp - GENESIS_TIMESTAMP.raw()) / 2;
```

Comment:
2 second block time

Reply:
But that's not a constant - it's a config value. :)

Reply:
Its technically a constant in the superchain. I feel like that value is very unlikely to change altho it would not be great to hardcode it. The source of truth for this value could be the `SuperchainConfig` since that is the location for superchain global config. The `SystemConfig` is the location for chain specific config.

Reply:
We don't hardcode it in the L2 Output Oracle. It's not constant per testing chain as well.